uaclient/api/u/pro/security/fix/_common/plan/v1.py

```diff
@@ -521,12 +524,36 @@ def _get_status(self) -> str:
         else:
             return str(FixStatus.SYSTEM_NON_VULNERABLE)
 
+    def _get_current_status(self) -> str:
+        system_status = self._get_expected_status()
+        if system_status == "error":
+            return "error"
+        if (
+            len(self.fix_steps) == 1
+            and isinstance(self.fix_steps[0], FixPlanNoOpStep)
+            and (
+                self.fix_steps[0].data.status == "system-not-affected"
+                or self.fix_steps[0].data.status == "cve-already-fixed"
+                or self.fix_steps[0].data.status == "cve-fixed-by-livepatch"
```

Comment:
just a style suggestion, but we can update this for a `in` check:
`self.fix_steps[0].data.status in ("system-not-affected", "cve-already-fixed", "cve-fixed-by-livepatch")`

Reply:
Looks better, updated!